packages/builders/src/messages/embed/Embed.ts

```diff
@@ -170,7 +155,9 @@ export class EmbedBuilder {
 	 * You can set a maximum of 25 fields.
 	 * @param fields - The fields to set
 	 */
-	public setFields(...fields: RestOrArray<APIEmbedField>): this {
+	public setFields(
```

Comment:
wheres clearFields?

Reply:
I figured I'd skip it in favor for `setFields([])` 🤔 do you feel it's explicitly needed?

Reply:
If it works, its probably fine